nnvm/tests/python/frontend/keras/test_forward.py

```diff
@@ -21,8 +21,14 @@ def verify_keras_frontend(keras_model):
     for layer in keras_model._input_layers:
         in_shapes.append(tuple(dim.value if dim.value is not None else 1 for dim in layer.input.shape))
     out_shapes = []
-    for layer in keras_model._output_layers:
-        out_shapes.append(tuple(dim.value if dim.value is not None else 1 for dim in layer.output.shape))
+
+    for oc in keras_model._output_coordinates:
+        layer = oc[0]
+        if isinstance(layer.output, list):
```

Comment:
I think using `layer.output_shape` make the code simpler.

Reply:
What I meant was something like as follows.  Doesn't it work?
```
if isinstance(layer.output_shape, list):
    out_shape = layer.output_shape[oc[2]]
else:
    out_shape = layer.output_shape
out_shapes.append(tuple(dim if dim is not None else 1 for dim in out_shape))
```

Reply:
if `layer.output` is a list, `layer.output_shape` is also a list, I think.  Anyway, the current one is fine with me.